### PR TITLE
Create build script

### PR DESCRIPTION
**Pull request by markwylde**
- Move the `index.html` from the `dist` folder to `src`
- Remove the `dist` folder from the repo
- Move out the build logic from the `package.json` to a `build.js` file
- Add css bundle

### CODE REVIEW

File: demo/build.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import fs from 'fs';
```
Comment (votemike) [inline demo/build.js:1]: This is an old-school build file....
Where's the Grunt/Gulp/Webpack 😄

File: demo/build.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+fs.mkdirSync('./dist')
+
+// Build CSS
+esbuild.build({
+  entryPoints: ['./src/index.css'],
+  bundle: true,
+  outfile: './dist/index.css',
+  sourcemap: 'inline',
+  watch: isWatching,
+  loader: {
```
Comment (votemike) [inline demo/build.js:19]: Is this needed for CSS?

**Review by votemike — APPROVED** [review]

File: demo/build.js (hunk 1 above)
Comment (markwylde) [reply demo/build.js:1]: Beautiful isn't it. Crazy to actually be able to see exactly how your build works :grin:

File: demo/build.js (hunk 2 above)
Comment (markwylde) [reply demo/build.js]: Ahh nice catch. Fixed in e7f2a2c9783efbe2e72f6ec5b4975a1a0ac24c5b.

File: demo/build.js (hunk 1 above)
Comment (votemike) [reply demo/build.js:1]: I mean... I don't know how the innards of esbuild work...

**Review by votemike — APPROVED** [review]

**Pull request merged**